lib/go/templates/template_utils.go

```diff
@@ -3,6 +3,8 @@ package templates
 import (
 	"fmt"
 	"strings"
+
+	_ "github.com/kevinburke/go-bindata"
```

Comment:
Is this import needed?

Reply:
No, but I was having a lot of difficulty with go modules and dependencies getting CI to pass properly, and this is the only way I was able to figure out how to make it work. 🤷